crowdfund/contracts/SmartPoolToken.sol

```diff
@@ -0,0 +1,131 @@
+pragma solidity ^0.4.0;
+
+import "./ERC20.sol";
+import "./Lockable.sol";
+
```

Comment:
Following ERC20 is good.
But these days people add another layer of standard token.
See [crowdsale](https://github.com/OpenZeppelin/zeppelin-solidity/blob/master/contracts/token/CrowdsaleToken.sol) token.